db.js

```diff
@@ -220,21 +220,30 @@ export class ActivityPubDB extends EventTarget {
     await tx.done()
   }
 
-  async searchNotes (criteria, { skip = 0, limit = DEFAULT_LIMIT } = {}) {
+  async searchNotes ({attributedTo} = {}, { skip = 0, limit = DEFAULT_LIMIT, sort = -1 } = {}) {
```

Comment:
Still facing the same issue mentioned in https://github.com/hyphacoop/reader.distributed.press/pull/8

![Screenshot 2024-04-30 at 3 47 11 PM](https://github.com/hyphacoop/reader.distributed.press/assets/68826419/e12bcd4d-0138-4c6a-9fc3-0c94064b04f8)

![Screenshot 2024-04-30 at 3 47 27 PM](https://github.com/hyphacoop/reader.distributed.press/assets/68826419/62428910-e15d-4a25-9b80-9d44ebf5a2da)